doc/nrf/libraries/others/secure_services.rst

```diff
@@ -7,7 +7,7 @@ Secure Services
    :local:
    :depth: 2
 
-Secure Services are functions implemented in the Secure Firmware (:ref:`secure_partition_manager`), but made available to be called from the Non-Secure Firmware.
+Secure Services are functions implemented in the firmware in Secure Processing Environment (:ref:`secure_partition_manager`), but made available to be called from firmware in Non-Secure Processing Environment.
```

Comment:
Any reason we don't use abbreviations here for SPE and NSPE?

Reply:
The reason was more trivial: with the SPM link being in parentheses, I had no easy way to add the abbreviations without adding additional information.

Reply:
Rephrased to: "Secure Services are functions implemented in the firmware in Secure Processing Environment (SPE), but made available to be called from firmware in Non-Secure Processing Environment (NSPE)."

The link to SPM is provided in the next paragraph and in the API section, so I don't think we really need it in the first sentence.